contracts/LiquidDemocracy.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.4.4;
+
+contract LiquidDemocracy {
+    ////////////////
+    // Data structures
+    ////////////////
+
+    // This is the type for a single option shown in a ballot
+    struct Proposal {
+        bytes32 name; // short name (up to 32 bytes)
```

Comment:
I have read that strings are very expensive in gas, we can keep it for now, however we need to keep it in mind and remove them as soon as possible. We can probably add a mapping on the frontend proposalId -> proposal name.

Reply:
probably an id, would be a good idea as well